notebooks/Heating Scheduler.ipynb

```diff
@@ -0,0 +1,147 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Heating Scheduler Example\n",
+    "\n",
+    "Once you retrieved your authentication information from the MIA app, you can setup the client and begin with an initial device/room discovery:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import logging\n",
+    "\n",
+    "from environs import Env\n",
```

Comment:
How does this work within the virtualenv? It was complaining about missing this dependency

Reply:
What did the venv complain about? I just purged and reinstalled the environment and could run the notebook without issues. Maybe you can try this, too and see if it fixes your issue:

```sh
pipenv uninstall --all
pipenv install --dev
pip install notebook
jupyter notebook
```

Reply:
I followed the steps you did and get

![image](https://user-images.githubusercontent.com/713525/111043221-acb1b480-8441-11eb-8579-fc633848e0a1.png)

Reply:
Even though I launched it from within pipenv it seems to not be aware of the venv I guess :/

Reply:
https://stackoverflow.com/a/47296960/50913

This worked though 

```
python -m ipykernel install --user --name=`basename $VIRTUAL_ENV`
```

And then changing the kernel in the notebook!

Reply:
I never had any problems with jupyter inside virtual envs before, so I am happy you found this fix! I will update the README accordingly.

Reply:
It could be that you had `environs` installed globally... Running `pip list | grep environs` would verify that.

Never used jupyter before, def is neat!

Reply:
Just checked, but it is not in my system packages. (I usually use [pyenv](https://github.com/pyenv/pyenv) for other python projects, so I don't have many packages installed there.)

jupyter really shines for protoyping and also showcasing code examples. You can also use it with ipython extensions like [autoload](https://ipython.org/ipython-doc/stable/config/extensions/autoreload.html) which makes it even more convenient!